packages/web-app-preview/src/components/Sources/MediaImage.vue

```diff
@@ -97,5 +130,6 @@ export default defineComponent({
 img {
   max-width: 80%;
   max-height: 80%;
+  cursor: move;
```

Comment:
this was oddly necessary, as panzoom looses the cursor representation while rotating the image